### DISCUSSION TITLE
How do I add bootstrap-icons?

### DISCUSSION THREAD

**Discussion by bbardsley** [thuliteio/doks#640] [Q&A] [answered]
Hello,

I was trying to add bootstrap-fonts with `npm i bootstrap-icons` and adding `@import "bootstrap-icons/font/bootstrap-icons.css";
` to app.scss. But somehow the font stil doesn't show up when adding `<i class="bi bi-youtube"></i>
` to a shortcode (and then calling the shortcode in a markdown file). What am I doing wrong?

Thanks in advance!

**Comment by h-enk** [comment, marked as answer]
Make sure to import the `scss` file to `./assets/scss/app.scss`:

```scss
/** Import Bootstrap Icons */
@import "bootstrap-icons/font/bootstrap-icons.scss";
```

And, don't forget to copy `./node_modules/bootstrap-icons/font/fonts/` to `./static/fonts/`

Then it should work:

![Snag_207aa551](https://user-images.githubusercontent.com/3902872/151988729-0a4a1970-21f7-410a-a671-d2bd8816deb3.png)

**Reply by bbardsley** [reply]
Thank you. I forgot to copy the font. You are very helpful. I appreciate it very much.

**Reply by technosprout** [reply]
Getting and odd error: "assets/scss/app.scss
 11:47  ✖  Unexpected extension ".scss" in @import  scss/at-import-partial-extension"

**Reply by h-enk** [reply]
And without the extension?

```scss
/** Import Bootstrap Icons */
@import "bootstrap-icons/font/bootstrap-icons";
```

**Comment by mdehollander** [comment]
I did the same (but instead of copying use the module.toml file to mount the node_modules dir), but I am running in a strange issue. The request woff file is surrounded by quotes and therefore not found: `http://localhost:1313/fonts/%22bootstrap-icons%22.woff2?24e3eb84d0bcaf83d77f904c78ac1f47`

Looking in the bootstrap-icons.scss file in the node_modules dir, I see there is a variables (with quotes): 
```
$bootstrap-icons-font: "bootstrap-icons" !default;
```

When I define this variable in _variables.scss without quotes, the url is correct:
```
$bootstrap-icons-font: bootstrap-icons
```

Anyone an idea what is going on here? The fix seems a bit weird to me, but hey, it does work.

**Comment by james-d-elliott** [comment]
Updated method to include the fonts:

```diff
Subject: [PATCH] feat: include bootstrap-icons fonts
---
Index: config/_default/module.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config/_default/module.toml b/config/_default/module.toml
--- a/config/_default/module.toml	(revision 0dfb77066bfd161d001af579867dcd16532d81fd)
+++ b/config/_default/module.toml	(date 1709887773417)
@@ -26,6 +26,10 @@
   source = "node_modules/@tabler/icons/icons"
   target = "assets/svgs/tabler-icons"
 
+[[mounts]]
+  source = "node_modules/bootstrap-icons/font/fonts"
+  target = "static/fonts"
+
 [[mounts]]
   source = "assets"
   target = "assets"
```
